foreshadow/transformers/base.py

```diff
@@ -560,9 +560,9 @@ def fit(self, X, y=None, **kwargs):
             self._set_to_empty = True
         else:
             self._verify_transformer(X, y, refit=True, **self.kwargs)
-        inject = _inject_df(self._transformer, kwargs.pop("full_df", None))
+        self._transformer.full_df = kwargs.pop("full_df", None)
```

Comment:
Can we drop a note in issues to say that we removed this and expect it to be implemented in ColumnInfoSharer.

Reply:
```suggestion
        self._transformer.full_df = kwargs.pop("full_df", None) . # this was removed and will be expected to be implemented in ColumnSharer
```